UnrealEngine.gitignore

```diff
@@ -43,6 +50,22 @@
 *.sdf
 *.VC.db
 *.VC.opendb
+.idea/
```

Comment:
IDE-specific rules should be grouped under the relevant template in `Global` as this is preferred to keep maintenance work manageable. Please keep the changes focused to rules specific to UnrealEngine.